.github/workflows/linuxMain.yml

```diff
@@ -18,6 +18,12 @@ jobs:
 
       - uses: actions/setup-node@v2-beta
         
+      - name: install Pandoc
+        run: sudo wget https://github.com/jgm/pandoc/releases/download/2.14.0.2/pandoc-2.14.0.2-1-amd64.deb && sudo dpkg -i pandoc-2.14.0.2-1-amd64.deb
```

Comment:
Can't you just use `sudo apt-get install pandoc`. Then you don't have to change this file when a newer pandoc version is released.